### PR TITLE
Fix: CI/CD failed

### PR DESCRIPTION
**Pull request by jialez0**

### CODE REVIEW

**Comment by sameo** [comment]
@jialez0 Could you please separate the github actions fixes from the actual code fixes?

**Comment by jialez0** [comment]
> @jialez0 Could you please separate the github actions fixes from the actual code fixes?

OK, I'll do this.

**Comment by sameo** [comment]
> > @jialez0 Could you please separate the github actions fixes from the actual code fixes?
> 
> OK, I'll do this.

Just to be clear: I meant having 2 separate commits for that.

File: .github/workflows/eaa_kbc.yml [hunk 1]
```diff
@@ -25,34 +25,47 @@ jobs:
     steps:
       - name: Install rats-tls dependencies
         run: |
-          apt-get update && apt-get install -y autoconf gcc g++ make wget git cmake \
+          sudo apt-get update;
```
Comment (sameo) [inline .github/workflows/eaa_kbc.yml:28]: Why do we need `sudo` here?

**Review by sameo — APPROVED** [review]

**Pull request merged**